dns_dhcp.rst force dhcp option (#70)
NethServer/nethsecurity-docs: dns_dhcp.rst

**Advanced settings**

``Force DHCP server start`` 

> The DHCP of NethSecurity, upon startup, checks if there are other DHCP servers on the network. 

Reviewer: ```suggestion
Upon startup, the DHCP server checks if there are other DHCP servers on the network. 
```

With this option disabled, the DHCP server won't be activated if another one is detected on the network.

> With this option enabled, the DHCP server will be started even if there are others within the network.

Reviewer: ```suggestion
If the force option is enabled, the DHCP server will be started even if there are other DHCP servers within the network.
```
